store/tikv/gcworker/gc_worker.go

```diff
@@ -546,14 +546,23 @@ func (w *GCWorker) deleteRanges(ctx context.Context, safePoint uint64) error {
 
 		err = w.sendUnsafeDestroyRangeRequest(ctx, startKey, endKey)
 		if err != nil {
-			return errors.Trace(err)
+			logutil.Logger(ctx).Info("[gc worker] delete range failed on range",
+				zap.String("uuid", w.uuid),
+				zap.Binary("startKey", startKey),
+				zap.Binary("endKey", endKey),
+				zap.Error(err))
+			continue
 		}
 
 		se := createSession(w.store)
 		err = util.CompleteDeleteRange(se, r)
 		se.Close()
 		if err != nil {
-			return errors.Trace(err)
+			logutil.Logger(ctx).Info("[gc worker] failed to mark delete range task done",
```

Comment:
Do we need to change the `GCUnsafeDestroyRangeFailuresCounter` to the type `CounterVec` and add a label (e.g: "send" and "save") to indicate the stage of failure occurred.

Reply:
Good idea